src/components/select/select.js

```diff
@@ -370,29 +367,32 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $par
         if (angular.isString(disabled)) {
           disabled = true;
         }
+
         // Prevent click event being registered twice
         if (isDisabled !== undefined && isDisabled === disabled) {
           return;
         }
         isDisabled = disabled;
         if (disabled) {
-          element.attr({'tabindex': -1, 'aria-disabled': 'true'});
-          element.off('click', openSelect);
-          element.off('keydown', handleKeypress);
+          element
+            .attr({'aria-disabled': 'true'})
+            .removeAttr('tabindex')
```

Comment:
Are we sure, that the template will always be cloned, so that removal of tabindex here is not going to remove "attr.tabindex" from the template? I think that we mustn't lose the original tabindex value set by user.